GPKG: Enforce LON_LAT order in auto mode

cportele opened the pull request:
Closes https://github.com/interactive-instruments/ldproxy/issues/959

Tested with daraa.gpkg (EPSG:4326, requires LON_LAT enforcement) and Weinlagen.gpkg (EPSG:25832, already LON_LAT, or more correctly, Easting/Northing).

azahnen reviewed (dismissed):
I would be fine with merging this since GeoPackage is the main format. But we also support plain SpatiaLite where I guess this should not be applied? That would mean we would have to set the force value in the schema crawler and then propagate it to additionalInfo.

cportele commented:
@azahnen - I have implemented the approach that you described. I have added the information about enforcing an axis order to SqlDialect. Please have a look.
